# Int32 Tuple ROS2 Message defintion
int32 x1
int32 x2
